specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/networkWatcher.json

```diff
@@ -3380,6 +3400,244 @@
+          "description": "Describes the threshold at which the connection monitor test configuration is considered successful."
+        }
+      },
+      "required": [
+        "name",
+        "protocol"
+      ],
+      "description": "Describes a connection monitor test configuration."
+    },
+    "ConnectionMonitorTCPConfiguration": {
+      "properties": {
+        "disableTraceRoute": {
+          "type": "boolean",
+          "description": "A value indicating whether traceroute is disabled."
+        },
+        "port": {
+          "type": "integer",
+          "description": "The destination port on which a TCP test is to be performed."
+        }
+      },
+      "required": [
+        "port"
+      ],
+      "description": "Describes a configuration for a TCP test."
+    },
+    "ConnectionMonitorICMPConfiguration": {
+      "properties": {
+        "disableTraceRoute": {
+          "type": "boolean",
+          "description": "A value indicating whether traceroute is disabled."
+        }
+      },
+      "description": "Describes a configuration for an ICMP test."
+    },
+    "ConnectionMonitorHTTPConfiguration": {
+      "properties": {
+        "method": {
+          "type": "string",
+          "description": "HTTP method.",
+          "enum": [
+            "Get",
+            "Post"
+          ],
+          "x-ms-enum": {
+            "name": "HTTPConfigurationMethod",
+            "modelAsString": true
+          }
+        },
+        "port": {
+          "type": "integer",
+          "description": "The destination port on which a HTTP test is to be performed."
+        },
+        "path": {
+          "type": "string",
+          "description": "The path to add to the provided URI."
+        },
+        "requestHeaders": {
+          "type": "array",
+          "description": "The HTTP request headers.",
+          "items": {
+            "$ref": "#/definitions/HTTPHeader"
+          }
+        },
+        "validStatusCodeRanges": {
+          "type": "array",
+          "description": "A list of acceptable HTTP status code ranges. For instance: [ \"100\", \"200-205\", \"3xx\" ]",
+          "items": {
+            "type": "string"
+          }
+        }
+      },
+      "description": "Describes a configuration for a HTTP test."
+    },
+    "ConnectionMonitorSuccessThreshold": {
+      "properties": {
+        "checksFailedPercent": {
+          "type": "integer",
+          "description": "Percentage of failed checks."
+        },
+        "roundTripTimeMs": {
+          "type": "number",
+          "description": "Maximum roundtrip time in milliseconds."
+        }
+      },
+      "description": "Describes the threshold at which a connection monitor test configuration is considered successful."
+    },
+    "ConnectionMonitorEndpointFilter": {
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [
+            "Include"
```

Comment:
>Include [](start = 13, length = 7)

nit - you may remove this. may get missed, when updating enum values in the future